ast/ddl.go

```diff
@@ -477,6 +477,8 @@ type ColumnOption struct {
 	AutoRandomBitLength int
 	// Enforced is only for Check, default is true.
 	Enforced bool
+	// Name is only used for Check Constraint name.
+	Name string
```

Comment:
How about renaming it to `ConstraintName`?

Reply:
good

Reply:
addressed